project-maintainers.csv

```diff
@@ -1353,7 +1353,7 @@ Sandbox, Lima, Akihiro Suda, NTT, AkihiroSuda, https://github.com/lima-vm/lima/b
 ,,Jan Dubois, SUSE, jandubois,
 ,,Anders F Björklund, Individual, afbjorklund,
 ,,Balaji Vijayakumar, Thoughtworks, balajiv113,
-Sandbox, ContainerSSH, Nikos Tsipinakis, CERN, tsipinakis, https://github.com/orgs/ContainerSSH/people
+Incubating, ContainerSSH, Nikos Tsipinakis, CERN, tsipinakis, https://github.com/orgs/ContainerSSH/people

```

Comment:
This seems the line for ContainerSSH, not for Lima

Reply:
Good catch. Thank.
Fat finger fault.